src/lib/stable/canister_methods/inspect_message.ts

```diff
@@ -1,6 +1,24 @@
 import { Context, decoratorArgumentsHandler, OriginalMethod } from '.';
 
-// TODO explain here in a jsdoc that the dev can get the raw args using argDataRaw
+/**
+ * Decorator to mark a method as the inspectMessage method entry point.
+ *
+ * The inspectMessage method entry point will be called just before a call to an update method entry point.
+ *
+ * Arguments to the update method entry point can be accessed using `argDataRaw`.
```

Comment:
if we are putting a reference to `argDataRaw` do we also want to put a reference to `methodName` and `acceptMessage` conversely if we aren't do the others should we also not do `argDataRaw` or is this too nit picking for release candidate